test/models/questionnaire_test.rb

```diff
@@ -205,6 +205,18 @@ class QuestionnaireTest < ActiveSupport::TestCase
     end
   end
 
+  context "#clean_negative_special_needs" do
+    should "return nil if special_needs field is None" do
+      questionnaire = create(:questionnaire, special_needs: "NoNE")
+      assert_nil questionnaire.special_needs
+    end
+
+    should "return nil if special_needs fiels is n/a" do
+      questionnaire = create(:questionnaire, special_needs: "N/a")
+      assert_nil questionnaire.special_needs
+    end
```

Comment:
Could you add another test here for if special_needs is something like the following?

```ruby
create(:questionnaire, special_needs: "None except for peanuts")
```

In this case, we wouldn't want the special needs to be stripped